Add some of the PypeIt CALSPEC flux standards

rremigio opened the pull request:
Added a few of the PypeIt CALSPEC flux standards to the stds folder. These are all WD stars with V_mag > 12, and wavelength coverage sufficient for very red KCRM setups.  All stars are observable from Keck II with a maximum elevation of > 45 degrees. 

I have not updated the spec_stats or stds_stats text files as I was unsure how to interpret some of the column names.

rremigio committed:
Add some of the PypeIt CALSPEC flux standards

MNBrod commented:
For the standards that already were in the pipeline, can you summarize what you did with the plots?

e.g. `bd02d3375.png` looks like a re-plot with extended coverage in the blue, but there's no corresponding change to the raw data for that target.

rremigio commented:
Regarding the plots that got replaced, this happened simply as a result of running the spec_stats.py script that was already in the standards folder, so whatever change between the old plots and the new ones is from there.

The script just sets log scaling on the y axis, and the x axis limits to min wavelength - 100 and min(12000 angstroms, wlmax) + 100. I can change this so its more consistent with the old plots going from ~3200 A to 1.1 micron (and update the .txt files too)

I have checked bd02d3375 specifically and it seems that the old plot for that object is likely based on a version of the spectrum that did not have the extended wavelength coverage (or it was probably trimmed to only have wavelengths within the KCWI wavelength range).

rremigio committed:
change the format of the standard star plots to look more like the old ones

change the markers to blue x, and fix  the x limits to be 3200 A to 11000 A. y limits are simply set to the maximum and minimum flux values within the KCWI wavelength range.

MNBrod commented:
Sounds good to me, thanks for the explanation. I think you're right, the last time we added to the standards we must have forgotten to run the stats.

Sending to Rosalie for final approval, then we'll merge it in.

rremigio committed:
update kcwi_stds_starlist.txt file

updated the information in the kcwi_stds_starlist.txt file.

Added stars in the standards folder that were previously unlisted, and added information on the WD stars added.

Removed a few existing standards that were the same as the WD stars.

rremigio committed:
Update spec_stats.txt

rremigio committed:
Update std_stats.txt

Last updates taking into account the standards I added. Also adds entries for missing stars.

rremigio committed:
added gd153

adds the calspec standard gd153. Updates the corresponding text files.

rmcgurk commented:
@rremigio I am horribly late in reviewing, but I have a question. Some of the plots that now list an extended range on the top don't have a new fits file added, so was the extra data outside of our plotting range just not counted before? For example, hd60753 and hz43 have extended ranges but no modified fits files are listed. Please explain.

rremigio commented:
@rmcgurk I checked the two stars you mentioned (hd60753 and hz43) as well as some other stars (e.g., hz2, gd71, bd26d2606), and it seems that the new plots I made just list the minimum and maximum wavelengths as read from the .fits table, while the old plots list the maximum and minimum wavelengths within the KCWI (Blue + Red) wavelength range, even if the standard star spectrum's wavelength range goes beyond well beyond that. 

Do you have a particular preference for what wavelength range should be listed in the plots?

rmcgurk commented:
@rremigio I am fine with the wavelengths listed, but I was trying to understand where the extra coverage came from without a new fits. In studying this more, I realized that the following stars are currently removed from the master branch kcwi_stds_starlist.txt for the following reasons (but it looks like I failed to remove their corresponding fits and pngs, so that is on me). 
- hd19445 is high proper motion (fine) and a variable star (not fine)
- hd60753 is at dec -50 and is too low
- hz29 is a variable star
- hz43 is a variable star
Could you remove the plots and fits for these, as well as their entries in kcwi_stds_starlist.txt? Then i will merge this in. 
Thanks!

rremigio committed:
remove hd19445, hd60753, hz29, and hz43

Rosalie's pre-merge changes

rremigio commented:
@rmcgurk Made the changes and have also removed the same four stars in the spec_stats.txt and stds_stats.txt files

rmcgurk commented:
@rremigio Excellent, thanks!
